### ISSUE TITLE
Does it still work? 

### ISSUE THREAD

**Issue by danyaljj** [james2doyle/google-form-styling#4]
Does the tool still work, or it's compatible with the older version of Google forms?

**Comment by james2doyle** [comment]
I don’t believe so. Someone in a past issue shared this https://codepen.io/JacobLett/pen/dOLBOr not sure if that works
